src/data/markdown/translated-guides/en/03 Using k6 browser/02 Running browser tests.md

```diff
@@ -90,7 +90,7 @@ $ k6 run script.js
 #
 # You can find an example of a hardened SECCOMP profile in:
 # https://raw.githubusercontent.com/jfrazelle/dotfiles/master/etc/docker/seccomp/chrome.json.
-docker run --rm -i grafana/k6:master-with-browser run - <script.js
+docker run --rm -i --cap-add=SYS_ADMIN -v $(pwd):/home/k6/screenshots grafana/k6:master-with-browser run - <script.js
```

Comment:
I believe [we no longer need](https://github.com/grafana/k6/pull/3340) to add `SYS_ADMIN`.

```suggestion
docker run --rm -i -v $(pwd):/home/k6/screenshots grafana/k6:master-with-browser run - <script.js
```